buildSrc/settings.gradle

```diff
@@ -0,0 +1,9 @@
+enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
Is `enableFeaturePreview` still needed for that? generally, as of Gradle 7.4.2, in the top-level project this works OOTB